doc/news/_2016-12-17_website_release.md

```diff
@@ -0,0 +1,118 @@
+# Website Release #
+
+- guid: 102b84a3-c41e-485c-8fe2-f12a24b3fbfd
+- author: Marvin Mall
+- pubDate: <TODO>
+- shortDesc: introduces new Elektra website & snippet sharing
+
+## Introduction ##
+
+With Elektra developing into a more and more reliable as well as
+popular system to manage system configurations, the demand for a
+better public appearance increases as well. For this reason, we
+are happy to be able to announce the release of our new website!
+
+The new website does not only give us a chance to better present
+ourselves to the open world, it also enables us to structure our
+project documentation better. We hope that this will make it easier
+for our users to get started with Elektra and all of its awesome
```

Comment:
We already have the awesome "full battery" ;)

What about adding quotes what people (including yourself) say? This would be a completely different story. Maybe even on the website itself? I'll send you an email about that later.

Nevertheless, there is a single occurrence of awesome (in the whole repo!), so it should be okay.